stack-lts-16.1.yaml

```diff
@@ -3,4 +3,4 @@ packages:
 - .
 
 extra-deps:
-- c14n-0.1.0.1@sha256:c56a513c1363d126ee704656b59d2e2af1cfe878587a97cb69ab0122b82e2d4d,1371
+- c14n-0.1.0.3
```

Comment:
Could you make sure that the sha256 hash is included everywhere?

Reply:
Sure